src/REPLMode.jl

```diff
@@ -337,6 +336,7 @@ function do_cmd!(tokens::Vector{Token}, repl)
 
     # Using invokelatest to hide the functions from inference.
     # Otherwise it would try to infer everything here.
+    cmd.kind == CMD_ACTIVATE    ? Base.invokelatest(      do_activate!, ctx, tokens) :
```

Comment:
A problem with putting this here is that it requires an active environment. Otherwise creating `ctx` will fail.

Reply:
Should be fixed now.